dydx/dydxPresenters/dydxPresenters/_v4/Portfolio/Components/dydxPortfolioPositionsViewPresenter.swift

```diff
@@ -119,7 +119,9 @@ class dydxPortfolioPositionsViewPresenter: HostedViewPresenter<dydxPortfolioPosi
         let item = positionsCache?[position.assetId] ?? dydxPortfolioPositionItemViewModel()
 
         let positionSize = abs(position.size.current?.doubleValue ?? 0)
+        let notionalValue = abs(position.valueTotal.current?.doubleValue ?? 0)
```

Comment:
hmmm i'm using `notionalTotal` here, what is `valueTotal`?

Reply:
```
val valueTotal = size * oraclePrice
set(valueTotal, modified, "valueTotal", period)
val notional = valueTotal.abs()
set(notional, modified, "notionalTotal", period)
```

i think we want `notionalTotal` here

Reply:
mm good catch. I was looking for notionalTotal earlier but autofill wasn't filling.

for our knowledge 
```
                            val valueTotal = size * oraclePrice
                            set(valueTotal, modified, "valueTotal", period)
                            val notional = valueTotal.abs()
                            set(notional, modified, "notionalTotal", period)
```

 addressed in

Reply:
mmm good catch, i was looking for notionalTotal earlier but it wasn't autopopulating addressed in ba3fd6ca4bffc71d425d4bb79fc1300d881fd39b